src/bun.js/test/jest.zig

```diff
@@ -2007,6 +2007,30 @@ fn consumeArg(
     args_idx.* += 1;
 }
 
+fn resolvePropertyPath(globalThis: *JSGlobalObject, obj: JSValue, path: []const u8) !?JSValue {
```

Comment:
This one should probably reuse `getIfPropertyExistsFromPath`, but I think we could ship without that for now